src/condor_utils/param_info.in

```diff
@@ -4910,6 +4910,18 @@ default=gstat --all --mpifile --gmond_ip=localhost --gmond_port=8649
 type=string
 tags=condor_config
 
+[GANGLIAD_RESET_METRICS_FILE]
+default=$(SPOOL)/metricsToReset.ganglia_metrics
```

Comment:
If you put something into Spool, should you also tell preen not to remove it?

Reply:
The file does not grow without bounds (i.e. it is not like a log file), there is only one such file per gangliad instance, and overall file size will be relatively small ... O(kbytes).   The problem is I could not think of a good algorithm for when preen can safely remove the file.... it could only be if condor_gangliad is shut down and never subsequently restarted, or if ganglia's database is purged.   Both cases are hard for preen to know :).  Just looking at the mtime or atime of the file is not a good algorithm; in a static pool (i.e. not a glidein pool), this file may need to stick around unmodified for months.  

So it becomes a decision of weighing the pros -vs- cons.  The pro of preen removing the file is we save a few kbytes of disk space.  The bad news of preen removing the the file when it should not is that metrics may not get reset when the gangliad is restarted.  I felt that the cons outweighed the pros, especially considering preen does not know when it is safe to remove.

If you agree, please resolve.  If not, please suggest an algorithm preen could use.

Reply:
Preen has been updated to check for metrics reset files in spool